components/scream/src/share/tests/common_physics_functions_tests.cpp

```diff
@@ -238,15 +238,28 @@ void run(std::mt19937_64& engine)
   qv0  = pdf_qv(engine);  // This is an input for mmr_tests, so it won't be modified by mmr tests
   // mmr_test1: For zero drymmr, wetmmr should be zero
   // mmr_test2: For zero wetmmr, drymmr should be zero
-  // mmr_test3: Compute drymmr from wetmmr0 and then use the result to compute wetmmr, which should be equal to wetmmr0
+  // mmr_test3: Compute drymmr from wetmmr0 and then use the result to compute wetmmr, which should be approximately
+  //            equal to wetmmr0
+
+  // mmr_test4: [to test mathematical properties of the function] Compute drymmr from wetmmr0 and then use the result to
+  //            compute wetmmr. "qv" should be in the form of 2^k+1 (1 is added as we have [1-qv] in the numerator or
+  //            denominator of this function), so that the results of wet->dry->wet are exactly
+  //            the same as the initial input mmr(wetmmr0). This is a mathematical property test, so physically invalid qv
+  //            values are also acceptable
```

Comment:
Did you mean "unphysical values"?

Reply:
Also, I would add a `TODO` comment, stating that if we ever add inputs checks in the conversion functions, this test might have to be removed. Just so that devs don't freak out if/when they add the check and start seeing tests failing... :)

Reply:
I have added a *Warning* to alert developers that it might fail if a check is added to the function. It is a little unsatisfactory as someone has to go into the code and read this comment but I am not sure of there is a better way to display this info.

Reply:
Yeah, I don't see any better way. The string `TODO` (all uppercase) is sometimes highlighted by some text editors, but that's about it, and it's not much more helpful than a normal warning. I think this is fine.